design_pim/overview.rst

```diff
@@ -0,0 +1,374 @@
+
+
+Summary
++++++++
+
+We just created our first module and made it available for other part of the application. You can now use it by requirering ``my_locale_logger``. We cannot use it right know as but we will see that in a minute.
+
+
+Create our first Backbone view
+==============================
+
+Basic view
+++++++++++
+
+Now that we know how to create a RequireJS module, let's create our first Backbone view.
+
+.. code-block:: javascript
+    :linenos:
+
+    // src/Acme/Bundle/CustomBundle/Resources/public/js/new_view.js
+    define(
+        ['backbone'],
+        function (Backbone) {
+            return Backbone.View.extend({
+
+            });
+        }
+    );
+
+
+This is the simplest view we can create using Backbone. It will result into an empty div if we add it to the DOM.
+
+The real stuff
+++++++++++++++
+
+Let's add some more interesting stuff to it:
+
+.. code-block:: javascript
+    :linenos:
+
+    // src/Acme/Bundle/CustomBundle/Resources/public/js/new_view.js
+    define(
+        ['backbone'],
+        function (Backbone) {
+            return Backbone.View.extend({
+                events: {
+                    'click': 'clicked'
+                },
+
+                render: function () {
+                    this.$el.html('<div>Hello world</div>')
+                },
+
+                clicked: function (event) {
+                    console.log(event);
+                }
+            });
+        }
+    );
+
+
+In this code we do three things:
+- We override the ``render`` method to add custom render logic. In this example, we add a div inside our view with a simple text inside it.
+- We declare an event listener to listen to the click events on our view to call the `clicked` method
+- The clicked method will log the DOM click event each time a click is triggered on our view.
+
+You can find more informations about Backbone views on the `dedicated documentation <http://backbonejs.org/#View>`_
+
+
+Register it
++++++++++++
+
+Now that we have a view, we need to register it in the ``requirejs.yml`` file
+
+.. code-block:: yaml
+
+    # src/Acme/Bundle/CustomBundle/Resources/config/requirejs.yml
+    config:
+        paths:
+            my_locale_logger: acmecustom/js/locale_logger
+            my_custom_view: acmecustom/js/new_view
+
+
+After cache clearing, asset dump and webpack build, you should be able to use your newly created view.
+
+Create our first form extension
+===============================
+
+Now that we can create a RequireJS module and a Backbone view, we would like to actually customize the PIM to add our own business logic on top of it.
+
+A little bit of history
++++++++++++++++++++++++
+
+As we are an open source company, our product can used and customized for a lot of different reasons and by a lot of different people (integrators, clients, contributors, technological partners). With this in mind, it was impossible to base our architecture on overrides to customize it.
```

Comment:
Fix typo : As we are an open source company, our product can **be** used and customized for a lot of different reasons and by a lot of different people (integrators, clients, contributors, technological partners). With this in mind, it was impossible to base our architecture on overrides to customize it.